pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -212,23 +215,27 @@ def set_epoch(self, x_axis_var):
                             delta_minus_var.astype(float) \
                             * np.float(si_conv.split('>')[0])
 
-                # Sometimes these are specified as arrays
-                if isinstance(delta_plus_var, np.ndarray) \
-                        and isinstance(delta_minus_var, np.ndarray):
-                    delta_time = (delta_plus_var
-                                  - delta_minus_var) / 2.0
-                else:  # And sometimes constants
-                    if delta_plus_var != 0.0 or delta_minus_var != 0.0:
-                        delta_time = (delta_plus_var
-                                      - delta_minus_var) / 2.0
-
-        if self.get_dependency(x_axis_var) is None:
             if ('CDF_TIME' in data_type_description) or \
                     ('CDF_EPOCH' in data_type_description):
-                xdata = cdflib.cdfepoch.unixtime(xdata)
-                xdata = np.array(xdata) + delta_time
                 if self._datetime:
-                    xdata = pds.to_datetime(xdata, unit='s')
+                    # Convert xdata to datetime
+                    new_xdata = cdflib.cdfepoch.to_datetime(xdata)
```

Comment:
This line should probably be surrounded by a `try` / `except(TypeError)` statement to catch the error in the Jan 4-31 2014 cindi data.  The question is what do we want the output to be?  Raise a more useful error message? Warn and drop the data?

Reply:
Warn and drop: something about "File has corrupted times, contact CDAWeb for them to fix it"

Reply:
Probably also worth adding a try/except to line 428, in case my branch for cdflib gets merged in.